### DISCUSSION TITLE
[projects beta] filter out closed issues

### DISCUSSION THREAD

**Discussion by ssadowski-uturn** [community/community#13493] [Projects and Issues] [answered]
The above is really all of it - provide an option to not show closed issues.

This is useful in grid view where I do not yet wish to archive the cards.

**Comment by BirgitPohl** [comment]
They have the status done. I think you can do this with `-status:done`. But I am not sure if you distinguish between "done" and "closed" and if Github distinguishes this as well.

**Comment by knyghty** [comment, marked as answer]
You can also use `-is:open`

**Reply by credomilliken** [reply]
Or ` -is:closed`

**Reply by 2percentsilk** [reply]
`-is:closed` should give you the view you're asking for!
